src/checkbox/Checkbox.tsx

```diff
@@ -48,9 +52,14 @@ export const Checkbox = forwardRefWithAs<
     value,
     checked,
     disabled,
+    isDisabled,
```

Comment:
Should we accept both props?

Reply:
I guess so, chakra also accepts both native input prop & their own isDisabled props..

Makes sense since these form elements accepts native html props & their custom props.

Reply:
They don't accept both 

https://github.com/chakra-ui/chakra-ui/blob/main/packages/input/src/input.tsx#L36